src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -148,7 +148,7 @@
       AutomationProperties.AutomationId="Project_Button_Update"
       Click="InstallButton_Clicked"
       Visibility="{Binding InstalledVersion, Converter={StaticResource NullToVisibilityConverter}, Mode=OneWay}"
-      IsEnabled="{Binding IsSelectedVersionInstalled, Converter={StaticResource NotNullOrTrueToBooleanConverter}}"
+      IsEnabled="{Binding IsAvailableForInstall, Converter={StaticResource NotNullOrTrueToBooleanConverter}}"
```

Comment:
Do we need `Mode=OneWay` here as well?
Did you test if keep switch back and forth by selecting and unselecting value also reflected on `Enabld/Disabled` mode of button?

Reply:
I added the `Mode=OneWay` and yes I tested that. Also found that this error happened in the Install button so I added the same conditional.